Test/Connect-RivetSession.Tests.ps1

```diff
@@ -0,0 +1,51 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+BeforeAll {
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    function ThenNoErrors
+    {
+        $Global:Error | Should -BeNullOrEmpty
+    }
+
+    function WhenConnectingRivet
+    {
+        param(
+            [String] $Database
+        )
+
+        Connect-RivetSession -Session $RTSession -Database $Database

```

Comment:
Only pass the `-Database` parameter if it has a value, otherwise we're never testing calling this function without it.